lib/llm-events/aws-bedrock/chat-completion-message.js

```diff
@@ -6,6 +6,8 @@
 'use strict'
 
 const LlmEvent = require('./event')
+const { tokenUsageAttributesExist, tokenUsageHeadersExist } = require('../aws-bedrock/utils')
```

Comment:
Maybe this could match the way we're requiring in genai, or that could be changed to match what we're doing here. 
```suggestion
const { tokenUsageAttributesExist, tokenUsageHeadersExist } = require('./utils')
```